support-frontend/package.json

```diff
@@ -197,5 +197,5 @@
 	"lint-staged": {
 		"./assets/**/**/*.{js,jsx,ts,tsx}": "eslint"
 	},
-	"packageManager": "pnpm@10.5.0"
+	"packageManager": "pnpm@10.9.0+sha256.ea53ab747ac8b7921de63b42eab1adf021e1fe6ded05f62f00885cc33a46118d"
```

Comment:
Was this change intentional? I think it should be fine as corepack will handle it, but in case of issues it might be surprising that it happened in this PR!

Reply:
Yes fair point, I did when prompted without really thinking about it!